tests/unit/pubsub.tcl

```diff
@@ -354,14 +354,23 @@ start_server {tags {"pubsub network"}} {
     }
 
     test "Keyspace notifications: hash events test" {
+        r config set hash-max-listpack-entries 1 ;# TODO: removed after hfe listpack is merged
```

Comment:
we should check for listpack and dict

Reply:
done with [`005abdd` (#13237)](https://github.com/redis/redis/pull/13237/commits/005abddbeb9ae3a04f84963d7dab06889f8b7d31)